apps/products-feed/src/pages/api/manifest.ts

```diff
@@ -32,7 +30,7 @@ export default wrapWithLoggerContext(
           dataPrivacyUrl: "https://saleor.io/legal/privacy/",
           extensions: [],
           homepageUrl: "https://github.com/saleor/apps",
-          id: "saleor.app.product-feed",
+          id: "saleor.app.product-feedd",
```

Comment:
**Question:** won't it break existing apps?

Reply:
I was changing the name for dev purposes - forgot to roll it back, thanks!

Reply:
@andrzejewsky instead of doing this, you can add `process.env.APP_MANIFEST_ID`

You can set it different on dev, staging and prod